modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryLocalServiceTest.java

```diff
@@ -300,6 +483,51 @@ private AccountEntry _addAccountEntry(int status) throws Exception {
 		return accountEntry;
 	}
 
+	private AccountEntry _addAccountEntry(String name, String description)
+		throws Exception {
+
+		AccountEntry accountEntry = AccountEntryTestUtil.addAccountEntry(
+			_accountEntryLocalService, name, description);
+
+		_accountEntries.add(accountEntry);
+
+		return accountEntry;
+	}
+
+	private AccountEntry _addAccountEntryWithEmailDomain(String emailDomain)
+		throws Exception {
+
+		AccountEntry accountEntry = AccountEntryTestUtil.addAccountEntry(
+			_accountEntryLocalService, new String[] {emailDomain});
+
+		_accountEntries.add(accountEntry);
+
+		return accountEntry;
+	}
+
+	private AccountEntry _addAccountEntryWithParentAccountEntryId(
+			long parentAccountEntryId)
+		throws Exception {
+
+		AccountEntry accountEntry = _accountEntryLocalService.addAccountEntry(
+			TestPropsValues.getUserId(), parentAccountEntryId,
+			RandomTestUtil.randomString(50), RandomTestUtil.randomString(50),
+			null, null, WorkflowConstants.STATUS_APPROVED);
+
+		_accountEntries.add(accountEntry);
+
+		return accountEntry;
+	}
+
+	private AccountEntry _addAccountEntryWithUser(User user) throws Exception {
+		AccountEntry accountEntry = _addAccountEntry();
+
+		_accountEntryUserRelLocalService.addAccountEntryUserRel(
+			accountEntry.getAccountEntryId(), user.getUserId());
+
+		return accountEntry;
+	}
+
```

Comment:
I think we can move all the _addAccountEntry* helper method to the util class `AccountEntryTestUtil`

Reply:
The reason I have them in this method is because the `_addAccountEntry()` method also adds the `AccountEntry` to the `_accountEntries` list for deletion during test cleanup.